src/libsync/connectionvalidator.cpp

```diff
@@ -315,8 +315,9 @@ bool ConnectionValidator::setAndCheckServerVersion(const QString &version)
     // We attempt to work with servers >= 5.0.0 but warn users.
     // Check usages of Account::serverVersionUnsupported() for details.
 
-#if QT_VERSION >= QT_VERSION_CHECK(5, 9, 0)
+#if QT_VERSION >= QT_VERSION_CHECK(5, 9, 4)
     // Record that the server supports HTTP/2
+    // QTBUG-64359 is fixed in >= Qt 5.9.4
```

Comment:
This comment does not make sense here.
This code snippet is only there to enable the propagator to have more parallelism if the connection is HTTP/2,   but it does not enable or disable HTTP/2 or workaround any qtbug.

The #if is there only because so it compiles because  QNetworkRequest::HTTP2WasUsedAttribute is a 5.9 feature.  (and therefore the #if check should stay 5.9.0 regardless of any other checks that enable HTTP/2).